test/unit/api/cpp/sort_black.cpp

```diff
@@ -65,6 +65,17 @@ TEST_F(TestApiBlackSort, operators_comparison)
   ASSERT_NO_THROW(d_solver.getIntegerSort() >= Sort());
 }
 
+TEST_F(TestApiBlackSort, getKind)
+{
+  Sort b = d_solver.getBooleanSort();
+  Sort dt_sort = create_datatype_sort();
+  Sort arr_sort =
+      d_solver.mkArraySort(d_solver.getRealSort(), d_solver.getIntegerSort());
+  ASSERT_EQ(b.getKind(), BOOLEAN_SORT);
+  ASSERT_EQ(dt_sort.getKind(), DATATYPE_SORT);
+  ASSERT_EQ(arr_sort.getKind(), ARRAY_SORT);
```

Comment:
Why does this not include tests for parameterized sorts (e.g., BV, FP) and abstract sorts?

Reply:
I can add.